### ISSUE TITLE
"groupClick" not working when using "groupToggleElement:"header""

### ISSUE THREAD

**Issue by roypardi** [olifolkerd/tabulator#828]
Using version 3.3

The `groupClick` callback is not working in my setup when I click on the group header. The `groupDblClick` does work. The group setup options I am using are below. Is there anything I am missing here?

```
    groupBy:"groupID",
    groupToggleElement:"header",
    groupStartOpen:false,
    groupClick: function(e, group){
        console.log('groupClick');
    },
    groupDblClick:function(e, group){
        console.log('groupDblClick');
    },
    groupVisibilityChanged:function(group, visible){
        manageGroupVisibility(group, visible);
    },
```

**Comment by roypardi** [comment]
Additional info:

I changed `groupToggleElement:"header"` to "arrow" and clicking anywhere except the "arrow" icon in the group, I now get the `groupClick` event.

Perhaps my expectation was wrong that a click in the header would be passed through to the `groupClick` function and not trapped by the "arrow" or "header" button functionality? If this is expected functionality perhaps add a note in the docs.

**Comment by olifolkerd** [comment]
Hey @roypardi,

Could you post a full copy of your table constructor so i can try and replicate your issue.

Cheers

Oli

**Comment by roypardi** [comment]
Hi – this is the gist of it:

```
$("#timeline-states-table").tabulator({
    selectable: "highlight",
    resizableColumns: false,
    movableColumns: false,
    allowReselection: true,
    addRowPos: "bottom",
    movableRows: true,
    height: "90%",
    index: "id",
    layout: "fitColumns",
    groupBy: "groupID",
    groupToggleElement: "arrow",
    // groupToggleElement: "header",
    groupStartOpen: false,
    groupClick: function (e, group) {
        console.log('groupClick');
    },
    groupDblClick: function (e, group) {
        //e - the click event object
        //group - group component
        console.log('groupDblClick');
    },
    groupVisibilityChanged: function (group, visible) {
        manageGroupVisibility(group, visible);
    },
    dataGrouping: function () {
        // console.log('dataGrouping');
    },
    dataGrouped: function (groups) {
        // console.log('dataGrouped');
    },
    groupHeader:function(value, count, data, group){
        return value + "<span style='color:#df0; margin-left:10px;'>(" + count + " item)</span>";
    },

    rowMoved: function (row) {
        states = $("#timeline-states-table").tabulator("getData");
        jsonData.states = states;
        saveProject();
    },
    cellEdited: function (cell) {
        // handle checkox as radio btn

        var obj = cell.getData();

        if (cell.getField() == "initialState") {
            var states = $("#timeline-states-table").tabulator("getData");
            states.forEach(function (element) {
                if (element.id == obj.id) {
                    element.initialState = true;
                } else {
                    element.initialState = false;
                }
            }, this);

            jsonData.states = states;
            $("#timeline-states-table").tabulator('updateData', jsonData.states);
            saveProject();
        } else {
            jsonData.states = $("#timeline-states-table").tabulator("getData");
            saveProject();
            updateStatesMenu();
        }
    },
    columns: [

        { formatter: playIcon, width: 20, align: "center", headerSort: false, editable: false, cellClick: function (e, cell) { stateSelectedHandler(cell) } },
        {
            title: "State", field: "stateName",
            align: "left", headerSort: false, width: 90, editor: "input", editable: editCheck
        },
        {
            title: "First", field: "initialState", width: 30, formatter: "tick", editor: true,
            align: "center", editable: true, headerSort: false, editable: editInitialStateCheck
        },
        { formatter: updateStatesIcon, width: 20, align: "center", headerSort: false, editable: false, cellClick: function (e, cell) { updateStateHandler(cell) } },
        { formatter: removeIcon, width: 20, align: "center", headerSort: false, editable: false, cellClick: function (e, cell) { deleteStateHandler(cell) } },
    ]
});
```

**Comment by olifolkerd** [comment]
Hey @roypardi 

Turns out there was an issue with the order of some of the click bindings that was preventing the ***groupClick*** event from firing when the group header was set to toggle visibility.

I have just pushed a fix for this to the 3.4 branch which will be released later today.

Cheers

Oli :)

**Comment by olifolkerd** [comment]
Tabulator version 3.4 has now been officially release!

You can find details of the new release in the [News Section](http://tabulator.info/news)

Full documentation of all the new features can be found [Here](http://tabulator.info/docs/3.4)

Cheers

Oli :)
